mirrors/views/mirrorlist.py

```diff
@@ -127,4 +242,92 @@ def find_mirrors_simple(request, protocol):
     proto = get_object_or_404(MirrorProtocol, protocol=protocol)
     return find_mirrors(request, protocols=[proto])
 
+
+def mail_mirror_admins(data):
+    template = loader.get_template('mirrors/new_mirror_mail_template.txt')
+    for mirror_maintainer in ['anton@hvornum.se', 'pitastrudl@gmail.com']:
```

Comment:
This can be looked up (queryied from the users db) instead of hardcoding.

Reply:
Yepp, will get to it once I figure out where they're stored :)

Reply:
Fixed :) Easier than I imagined.